Transcripción final de casos especiales 2_5_2 diapositiva 10 hasta 20

joseucr opened the pull request:
Correspondiente a José Armando León S

joseucr committed:
Transcripción final de casos especiales 2_5_2 diapositiva 10 hasta 20, correspondiente a José Armando León S

DiegoAlfaroUCR reviewed (changes requested):
Tiene hasta el 19 de junio para aplicar las siguientes correcciones de formato y recibir la nota completa. Para esto edite el archivo y suba los commits al mismo branch desde el que realizó el pull request. Preferiblemente le da a la opción "Re-request review". Si tiene alguna consulta no dude en escribirme: diego.alfarosegura@ucr.ac.cr

Reglas de formato en sección "Sobre la transcripción" de documentación del repo.

Correcciones:
- No le renderizaron las figuras ni algunas fórmulas correspondientes a la diapositivas 11, 12 y demás. Desconozco porqué utilizó html y la sintaxis de html, por lo que no sé que error cometió en esas líneas donde instancia la imagen. (Si este error no se visualiza desde su computador al iniciar la documentación con MkDocs su pull-request, me indica para ver si es un problema mío).
- Coloque los ejemplos (como el de la diapositiva 14) en bloques de ejemplos según se define en sección "Sobre la transcripción".
- No le renderizó correctamente la nota de la diapositiva 17.

joseucr committed:
Update 2_5_2_casos_especiales_Corregido_B94249.md

Se realizaron la correcciones solicitadas, al correr el comando mkdocs serve se renderiza correctamente la pagina en el navegador web

DiegoAlfaroUCR reviewed (changes requested):
Corrigió bien las imágenes y la nota, pero no incluyó los enunciados de los ejemplos en bloques de formato ejemplo. Puede ver este formato en la sección "Sobre la transcripción" en el MkDocs.

joseucr committed:
Update 2_5_2_casos_especiales.md

Se aplicaron los cambios sugerido por el asistente, ahora los bloques de ejemplo y la respuesta se presentan como se solicita
